lightautoml/pipelines/features/text_pipeline.py

```diff
@@ -34,19 +34,11 @@
 
 
 class NLPDataFeatures:
-    """
-    Class contains basic features transformations for text data.
-    """
+    """Class contains basic features transformations for text data."""
```

Comment:
write about "default parameters for nlp pipeline constructor."

Reply:
Create task on the kanban board.